acceptance/suites/pre_suite/puppet3_compat/71_install_simmons_module.rb

```diff
@@ -1,4 +1,4 @@
-simmons_version = '0.3.1'
+simmons_version = '0.3.2'
```

Comment:
Thanks.  Was a small enough change that I just force-push updated to `0.3.3` in bba0571.